en/docs/basics_topics/optimizer_in_function_config.md

```diff
@@ -89,13 +39,17 @@ def get_eval_config():
   return config
 
 @flow.global_function(type="predict", get_eval_config())
-def eval_job():
-  # build up NN here
+def eval_job() -> tp.Numpy:
+  # build up neural network here
 ```
 
 ### Configuration for training
 
-Same as above, we decorate the `train_job()` function with `@flow.global_function` then we get a job function for training. We can set the optimizer, learning rate and hyperparameters inside the job function. 
+If you specify the `type` parameter of `@flow.global_function` to be `train`, you can get a job function for training.
+
+In the following code, `train_job` is the job function used for training and it is configured with the default `function_config` (so no parameter is passed to `function_config`).
+
```

Comment:
so no parameter is passed to `function_config` -> so there is no parameter passed to `function_config`